.github/workflows/ci.yml

```
@@ -16,6 +16,21 @@ env:
   PHP_MIN: '7.4'
 
 jobs:
+  entrypoint-bc-promise:
+    name: Entrypoint BC promise
+    runs-on: 'ubuntu-24.04'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup PHP
+        uses: ./.github/composite-actions/setup-php
+        with:
+          version: 5.6
```

```suggestion
          version: 7.0
```

please don't retrofix this because of https://3v4l.org/7ESuc#v5.6.40 - the https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8229 CS fixing would be then harder

the idea is PHP 5.6 is already unsupported (by the entry file) for a long time and the real usage is really super low

---

It's up to @keradus - his will was to support 5.6 and I believe the lost support for it was unintentional. If we want to change BC promise to 7.0 then I am fine with it, but it's not up to me.

---

The PHP 5.6 support was removed 3 years ago - https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/commit/60a618f52d5521fc92583989e30e3a9b577fb9bf - it might be unintentional, but 3 years we did not received any report. Users are simply not using PHP 5.6 anymore and the users who do, they already know "the syntax error behaviour". So IMO the value added returning support after 3 years is negative as it will complicate CI.

---

@mvorisek I know it and I agree with you, I just said it's not my decision 🙂.

---

as shared in my original msg ([ref](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8229#discussion_r1783512152)), we should align which versions we shall support for entry point.
I'm OK to have 7.0+